### DISCUSSION TITLE
Welcome to samosabucket Discussions!

### DISCUSSION THREAD

**Discussion by aannirajpatel** [aannirajpatel/samosabucket#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by joshishaswat** [comment]
Hello, my name is Shas and I am one of the co-founders of Samosabucket. I am studying comp sci at UNC Chapel Hill and took on this project to grow our platform to host ghost kitchens. I hope to add a couple features to this project including further customizations during the ordering process (we all have diff spice 🔥  tolerances).
